layers/+tools/eaf/packages.el

```diff
@@ -0,0 +1,219 @@
+;;; packages.el --- eaf layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2020 Sylvain Benner & Contributors
+;;
+;; Author: Daniel Nicolai <dalanicolai@daniel-fedora>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defconst eaf-packages
+  '((eaf :location (recipe
+                    :fetcher github
+                    :repo  "manateelazycat/emacs-application-framework"
+                    :files ("*")))))
+
+(defun eaf/init-eaf ()
+  (use-package eaf
+    ;; (eaf-bind-key clear_focus "C-f" eaf-browser-keybinding)
+    ;; (define-key eaf-mode-map (kbd "<escape>") 'eaf-proxy-clear_focus)
+    :defer t
+
+    :init
+    (spacemacs/declare-prefix "aa" "application-framework")
```

Comment:
I think these bindings should rather go under application/tools/web services, there should already be a prefix bound for this in the web search layer.

Reply:
Initially I updated the keybindings to this prefix, however, without the web search layer installed I still had to declare the prefix myself. On the other hand, I did not like the combination (too complicated). So no i moved everything under `aa` (and `aab`) i.e. `application`-`application-framework(-browser)` prefixes, as the browser is an application-framework application in the end anyway. (and typing `aa` is justnicer than typing `aw`).

Reply:
aa would be fine with me, however we should then also change the tag, as this layer would then do much more than just web-service related bindings i.e. I have seen functions accessing cameras for example.

I'll look up a proper tag when I am merging it.